### PR TITLE
fix: linux package error

### PR DESCRIPTION
**Pull request by xiaoluSF**

### CODE REVIEW

File: packages/neuron-wallet/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "package-mwl": "electron-builder -mwl",
     "package-win": "electron-builder --win --x64",
     "package-mac": "electron-builder --mac",
-    "package-linux": "electron-builder --linux",
+    "package-linux": "./script/installrpm.sh && electron-builder --linux",
```
Comment (classicalliu) [inline packages/neuron-wallet/package.json]: This script seems only works for macOS, what will happen if I run this on windows and linux ?

Comment (ashchan) [reply packages/neuron-wallet/package.json]: How about this?

* Add the full `package` script to `/script/package`.
* Only support macOS and Linux (let's forget running this on Windows).
* From that script, check if `rpm` is installed, if not, prompt the user to install.
* Allow passing `PLATFORM=linux` (just example) to that script, so we can leave only one `package` script in `package.json`.

File: packages/neuron-wallet/script/package.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        mwl)
+        electron-builder -mwl
+        ;;
+        *)
+        echo "Please add platform parameters like 'mac' 'win' 'linux' 'mwl' ."
+        ;;
+        esac
+}
+
+function checkandinstallrpm() {
```
Comment (classicalliu) [inline packages/neuron-wallet/script/package.sh]: Using snake_case to split function name

File: packages/neuron-wallet/script/package.sh [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                    yumdnf="dnf"
+                fi
+                echo "Please run '$yumdnf install -y rpm' to intall rpm"
+                ;;
+                *)
+                exit 1
+                ;;
+            esac
+        else
+            echo "Other OS: ${OS}"
```
Comment (classicalliu) [inline packages/neuron-wallet/script/package.sh]: Reminder user to install rpm

File: packages/neuron-wallet/script/package.sh [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        fi
+    else
+        package $1
+    fi
+}
+
+cd packages/neuron-wallet/
+getplatform $1
```
Comment (classicalliu) [inline packages/neuron-wallet/script/package.sh]: Insert new line at the end of file

File: packages/neuron-wallet/script/package.sh [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+        ;;
+        esac
+}
+
+function checkandinstallrpm() {
+    echo "Check rpm..."
+    if ! [ -x "$(command -v rpm)" ] ; then 
+        OS=`uname -s`
+        if [ ${OS} == "Darwin"  ];then
+            echo "Please run 'brew install rpm' "
```
Comment (classicalliu) [inline packages/neuron-wallet/script/package.sh]: Also add `to install rpm` at the end as following

File: packages/neuron-wallet/script/package.sh [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+        electron-builder --win --x64
+        ;;
+        linux)
+        electron-builder --linux
+        ;;
+        mwl)
+        electron-builder -mwl
+        ;;
+        *)
+        echo "Please add platform parameters like 'mac' 'win' 'linux' 'mwl' ."
```
Comment (ashchan) [inline packages/neuron-wallet/script/package.sh]: I suggest by default we build all. That means `mwl` is not required but set as default, so when we run it without any parameter just build all three platforms.

**Comment by classicalliu** [comment]
And update README to notice developers this command only works in macOS and linux.

**Comment by ashchan** [comment]
@classicalliu Please check the updated script and see if it's good to go.

File: packages/neuron-wallet/script/package.sh [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+        mac)
+        electron-builder --mac
+        ;;
+        win)
+        electron-builder --win --x64
+        ;;
+        linux)
+        electron-builder --linux
+        ;;
+        mwl)
```
Comment (classicalliu) [inline packages/neuron-wallet/script/package.sh]: This option is redundant, please remove it.

**Comment by classicalliu** [comment]
> @classicalliu Please check the updated script and see if it's good to go.

Script works well.

**Pull request merged**